cspell.json

```diff
@@ -1,6 +1,9 @@
 {
 	"version": "0.2",
 	"language": "en_GB",
+	"words": [
+		"rcompare"
+	],
```

Comment:
For new words please add them to dictionary4.txt. Other than that I think the PR looks good